java/pending_code_changes.txt

```diff
@@ -1 +1,11 @@
+Pending metadata changes:
+* Metadata fixes for BY, KR, NA
```

Comment:
No strong opinions - but we usually use '-' instead of '*' (release notes aren't meant to be markdown although they do end up in PR descriptions). See https://github.com/googlei18n/libphonenumber/blob/642590c597a49b526e632dc90770fc74b8a794ea/java/pending_code_changes.txt

Optional